packages/slice-machine/pages/onboarding.tsx

```diff
@@ -198,17 +200,22 @@ export default function Onboarding(): JSX.Element {
         )}
       </Flex>
 
-      <Flex
-        sx={{
-          gridArea: "content",
-          alignItems: "center",
-          justifyContent: "center",
-          alignContent: "center",
-          flexDirection: "column",
-        }}
-      >
-        {STEPS[state.step] ? STEPS[state.step] : <Spinner />}
-      </Flex>
+      {STEPS.map((Component, i) => (
+        <Flex
+          key={`step-${i + 1}`}
+          sx={{
+            gridArea: "content",
+            alignItems: "center",
+            justifyContent: "center",
+            alignContent: "center",
+            flexDirection: "column",
+            opacity: i === state.step ? "1" : "0",
+            zIndex: i === state.step ? "9999" : "-1",
```

Comment:
Why z -index is needed if the element is not visible ?

Reply:
As is, the element is invisible but still present in the DOM, z-index makes sure you click on the current element no matter the position of the others. There are probably other strategies (display: none) but I know this one would work if we wanted to implement CSS animations int he slideshow

Reply:
Ok not very sexy but good enough

Reply:
Yeah! I can look for something else

Reply:
Quoting an article I just read: "display: none has been the favorite solution to hide elements for many years, but it’s been superseded by more flexible, animatable options. It’s still valid, but perhaps only when you want to permanently hide content from all users. transform or opacity are better choices when considering performance."

I can maybe change the zIndex to pointer-events: none if that seems more appropriate to you